bgpd/bgp_fsm.c

```diff
@@ -710,6 +710,7 @@ int
 bgp_start (struct peer *peer)
 {
   int status;
+  afi_t afi;
```

Comment:
afi has an automatic storage duration. The value of such variable is not determined by C standard. So the code behavior is undefined in such cases.
So it could be AFI_IP or AFI_IP6. So you need to initialize the variable to something, which is not equal to AFI_IP or AFI_IP6.

Reply:
fixed it